scripts/packaging/publish_release.py

```diff
@@ -85,7 +85,7 @@
 
 if not releaseExists:
     print('Creating release "%s" with tag "%s" on commit "%s"' % (title, tag, options.commit))
-    draft = True if options.tag else False
+    draft = False if tag.startswith('nightly_') else True
```

Comment:
`tag` is a scoped variable, defined in if condition before. I am surprised it used to work.

Maybe we should define `tag` as a public variable, just `tag = None` somewhere in the beginning.